pom.xml

```diff
@@ -94,6 +94,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.jahia.bundles</groupId>
+            <artifactId>org.jahia.bundles.jspapiusage.repackaging</artifactId>
+            <version>8.1.3.0</version>
+            <scope>compile</scope>
```

Comment:
Did you try to put the scope to: `provided` ?
By using the `compile` scope it will make this dependency embedded in the jar, and it's not correct since it's provided into the OSGI system as a bundle already.